builder.go

```diff
@@ -73,20 +73,27 @@ type kubeBuilder struct {
 	schema    string
 }
 
-func splitServicePortNamespace(hpn string) (service, port, namespace string) {
-	service = hpn
-
+func splitServicePortNamespace(hpn string) (string, string, string) {
```

Comment:
I think it better the previous one that the returns has a clear name
Now someone needs to look at the function implementation first to know the returns

Reply:
I don't like named returns, I feel like it makes the code more difficult to read and is not a feature that is used consistently. Writing functions in a consistent style makes reading the codebase easier and you know what to expect.

Reply:
I agree with @luthfikw, also when you're contributing to an open source project, one should respect the original code signature and authors' written code. That's why you should revert the change on the function return parameters 🙏

Reply:
I'll change it back because it is your repository, but I still stand by my point. Consistency is important and this style is not used anywhere else in the project. It is also part of the community's job to try and improve things. I thought this was an improvement.